en/editor-guidelines.md

```diff
@@ -188,9 +188,9 @@ Full instructions on adding images is available in [Author Submission Instructio
 Similarly to Images, all data files should be stored on the site (not linked externally - for sustainability purposes). All data should be stored in the 'assets' directory, using the same rules as above, but authors should feel free to use a description for their data file that reflects what it is:
 
  - `/assets/LESSON-SLUG/Louvre-Paintings-1.csv`
- 
+
  Occasionally, large assets used by particular lessons might be too large to be stored in our GitHub repository. If this is the case, we recommend that authors upload their assets to [Zenodo](https://zenodo.org/) for archiving and then provide the lesson editor with the DOI generated by Zenodo for linking from within the lesson. In the event that such datasets might already exist in an institutional repository, we still recommend uploading the version of the dataset used in The Programming Historian to Zenodo for the purposes of consistent use across all of our lessons.
```

Comment:
```suggestion
 Occasionally, large assets used by particular lessons might be too large to be stored in our GitHub repository. If this is the case, we recommend that authors upload their assets to [Zenodo](https://zenodo.org/) for archiving and then provide the lesson editor with the DOI generated by Zenodo for linking from within the lesson. In the event that such datasets might already exist in an institutional repository, we still recommend uploading the version of the dataset used in *The Programming Historian* to Zenodo for the purposes of consistent use across all of our lessons.
```

Reply:
Hello @rivaquiroga. I have suggested some text for a new section in the Author Guidelines titled "What happens after your lesson is published?”, briefly outlining our Lesson Maintenance workflow and explaining that in case of bugs we may contact authors.